open-metadata-implementation/access-services/governance-engine/governance-engine-api/src/main/java/org/odpi/openmetadata/accessservices/governanceengine/api/events/GovernanceEngineEvent.java

```diff
@@ -5,6 +5,7 @@
 import com.fasterxml.jackson.annotation.JsonAutoDetect;
 import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
 import com.fasterxml.jackson.annotation.JsonInclude;
+import lombok.EqualsAndHashCode;
```

Comment:
NOTE this was included an experiment and made a commit on local branch. The changes were reversed in a later commit, so there is no lombok dependency until agreed or not (per asset catalogue omas)